### ISSUE TITLE
Home Assistant: Wrong URL to HA instance shown

### ISSUE THREAD

**Issue by spacegaier** [stanford-oval/almond-server#197]
After successful integration of Almond and HA, this is listed in the "My Skills" panel:

![grafik](https://user-images.githubusercontent.com/114137/117860853-0fcc9580-b291-11eb-8143-6e58da83c7cb.png)

The location "http://supervisor/homeassistant" however is wrong. Looks like a dev placeholder 😄.

**Comment by gcampax** [comment]
The location is correct, in fact. That's the URL used by add-ons to connect to Home Assistant from inside the docker contaienr network. We should probably hide it though if it is confusing.

**Comment by gcampax** [comment]
Again, in the code the location is correct - as you noted, the one in the addon wins. There is a DNS configuration in the supervisor that makes those URLs work. This is only a matter of displaying the right URL - which would need to come from Home Assistant's external URL then.

**Comment by spacegaier** [comment]
Yes, I understand that technically this URL is correct for the internal workings, but it's confusing for the end user (since it won't resolve outside of the internal docker containers). 

We are on the same page here 👍.
